repair/repair.cc

```diff
@@ -271,6 +272,14 @@ static std::vector<gms::inet_address> get_neighbors(database& db,
                     "part of the supplied list of hosts to use during the "
                     "repair (%s).", me, others, hosts));
         }
+    } else if (!ignore_nodes.empty()) {
+        std::vector<gms::inet_address> nodes;
+        nodes.swap(ret);
+        for (auto& node: nodes) {
+            if (!ignore_nodes.contains(node)) {
+                ret.push_back(node);
+            }
```

Comment:
nit: the intent would be clearer to the reader with `std::remove_if`

Reply:
I am not a fan of the  the remove_if + erase combination, but I can switch to it.